spec/hbw/features/bp_form/dynamic_conditions_by_select_table_spec.rb

```diff
@@ -32,6 +32,16 @@
     expect(page).to have_content 'Dependent file_upload'
     expect(page).to have_content 'Dependent radio_button'
 
+    in_group_with_label('Dependent submit_select group') do |group|
+      expect(group).to have_content 'Button 1'
+      expect(group).to have_content 'Button 2'
+    end
+    in_group_with_label('Dependent submit_select buttons group') do |group|
+      expect(group).to have_content 'Neutral button'
+      expect(group).to have_content 'Button to be disabled'
```

Comment:
I think you should check that this button is not disabled since it's always present.